tg/models.py

```diff
@@ -217,6 +226,114 @@ def copy_msgs_text(self) -> bool:
         copy_to_clipboard("\n".join(buffer))
         return True
 
+    def get_chat_info(self, chat: Dict[str, Any]) -> Dict[str, Any]:
+        chat_type = get_chat_type(chat)
+        if chat_type is None:
+            return {}
+
+        info = {}
+
+        if chat_type == ChatType.chatTypePrivate:
+            user_id = chat["id"]
+            user = self.users.get_user(user_id)
+            user_info = self.users.get_user_full_info(user_id)
+            status = self.users.get_status(user_id)
+            info.update(
+                {
+                    chat["title"]: status,
+                    "Username": user.get("username", ""),
+                    "Phone": user.get("phone_number", ""),
+                    "Bio": user_info.get("bio", ""),
+                }
+            )
+
+        if chat_type == ChatType.chatTypeBasicGroup:
+            group_id = chat["type"]["basic_group_id"]
+            result = self.tg.get_basic_group_full_info(group_id)
+            result.wait()
```

Comment:
not directly related to this PR, but the way we calls tdlib is urly:
```
result = self.tg.get_basic_group_full_info(group_id)
result.wait()
chat_info = result.update
```
we can have much better way of doing it, without wasting 2 lines for each tdlib call:
 `chat_info = self.tg.get_basic_group_full_info(group_id)`

our tdlib module should hide this complicity